.github/workflows/_ci.yaml

```diff
@@ -171,7 +180,7 @@ jobs:
       - build-jax
       - build-triton
       - build-equinox
-      - build-maxtext
+      - build-upstream-maxtext
```

Comment:
I think we want to publish both so two rows, similar to how we have `build-rosetta-pax` and `build-upstream-pax` here.